src/containers/Home/Home.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { render, screen, fireEvent } from "../../testUtils";
+import { Home } from "./Home";
+
+describe("Home", () => {
+  test("Should allow user to enter and submit valid postal code", async () => {
+    const { history } = render(<Home />);
+
+    const input = await screen.findByRole("textbox", {
+      name: /please enter your postal code/i,
+    });
+    fireEvent.change(input, { target: { value: "K2T0E5" } });
+    expect(input.value).toBe("K2T0E5");
+
+    const submit = screen.getByRole("button", { name: /submit/i });
+    fireEvent.click(submit);
```

Comment:
Minor, you should move to using `user-event` for interacting with dom nodes: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event